### PR TITLE
add html and css styles

### PR DESCRIPTION
**Pull request by IrinaSendetskaya**
check please my html and css files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+<meta charset="utf-8">
+<link href="style.css" rel="stylesheet">
+</head>
+
+<body>
+
+<h1>todos</h1>
+
+<div class="table">
+	<div class="div_first"><label> ˅ </label><em> What needs to be done?</em></div>
```
Comment (nastabulak) [inline index.html]: please update class names, no need to use word 'div'

File: index.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	<div class="div_second"><input type="checkbox" checked>to learn HTML</div>
+	<div class="div_3"><input type="radio">to learn CSS</div>
+	<div class="div_3"><input type="radio">to learn JS</div>
+
+	<div class="div_bottom">
+		<div class="div_bottom_1">2 items left</div>
+		<div class="div_bottom_bordered">All</div>
+		<div class="div_bottom_2">Active</div>
+		<div class="div_bottom_2">Completed</div>
```
Comment (nastabulak) [inline index.html]: please change class names to have it understandable for others

File: index.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+<head>
+<meta charset="utf-8">
+<link href="style.css" rel="stylesheet">
+</head>
+
+<body>
+
+<h1>todos</h1>
+
+<div class="table">
```
Comment (nastabulak) [inline index.html]: is it really table?;)

File: style.css [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+border-bottom: 1px solid grey;
+padding: 10px 1px 10px 10px;
+}
+
+.div_3{
+border-bottom: 1px solid grey;
+padding: 10px 1px 10px 10px;
+}
+.div_bottom_1{
+    display:inline-block;
```
Comment (nastabulak) [inline style.css:38]: you can create separate classes to include common styles and add them to elements.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+.div_first{
+color:grey;
+border-bottom: 1px solid grey;
+padding: 10px 1px 10px 10px;
+}
+.div_second{
+text-decoration:line-through;
+color:grey;
```
Comment (nastabulak) [inline style.css]: don't forget about inheritance of styles.

**Review by nastabulak — CHANGES_REQUESTED** [review]

File: index.html [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+<h1>todos</h1>
+
+<div class="container_tasks">
+	<div class="new_task"><label> ˅ </label><em> What needs to be done?</em></div>
+	<div class="task_completed"><input type="checkbox" checked>to learn HTML</div>
+	<div class="task_active"><input type="radio">to learn CSS</div>
+	<div class="task_active"><input type="radio">to learn JS</div>
+
+	<div class="container_filter">
```
Comment (nastabulak) [inline index.html]: filters is enough

File: index.html [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+<head>
+<meta charset="utf-8">
+<link href="style.css" rel="stylesheet">
+</head>
+
+<body>
+
+<h1>todos</h1>
+
+<div class="container_tasks">
```
Comment (nastabulak) [inline index.html]: tasks is enough

File: index.html [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+	<div class="new_task"><label> ˅ </label><em> What needs to be done?</em></div>
```
Comment (nastabulak) [inline index.html]: task_new  for consistency

File: index.html (hunk 1 above)
Comment (IrinaSendetskaya) [reply index.html]: fixed

File: index.html (hunk 2 above)
Comment (IrinaSendetskaya) [reply index.html]: changed

File: index.html (hunk 3 above)
Comment (IrinaSendetskaya) [reply index.html]: changed

File: style.css (hunk 4 above)
Comment (IrinaSendetskaya) [reply style.css:38]: changed

File: style.css (hunk 5 above)
Comment (IrinaSendetskaya) [reply style.css]: changed

File: index.html (hunk 6 above)
Comment (IrinaSendetskaya) [reply index.html]: changed

File: index.html (hunk 7 above)
Comment (IrinaSendetskaya) [reply index.html]: changed

File: index.html (hunk 8 above)
Comment (IrinaSendetskaya) [reply index.html]: changed

**Review by nastabulak — APPROVED** [review]

**Pull request merged**